packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -368,6 +368,16 @@
             "type": "string",
```

Comment:
Shall we add the definition in the aws-toolkiet-common pkg?

Reply:
Yes, will do